models.py

```diff
@@ -260,23 +269,28 @@ class PayDbTransaction(db.Model):
 
     id = db.Column(db.Integer, primary_key=True)
     token = db.Column(db.String(255), unique=True, nullable=False)
-    timestamp = db.Column(db.Integer)
+    date = db.Column(db.DateTime())
     action = db.Column(db.String(255), nullable=False)
     sender_token = db.Column(db.String(255), db.ForeignKey('user.token'), nullable=False)
+    sender_id = db.Column(db.Integer)
```

Comment:
you should not need this, you can use `sender_token`

Reply:
i cant figure out how.

Reply:
maybe i got something about this.... still testing..

Reply:
so created another filter without having to create new columns for the sender_id and recipient_id.